lib/rules/function-allowed-list/README.md

```diff
@@ -73,3 +73,46 @@ a {
     -moz-linear-gradient(45deg, blue, red);
 }
 ```
+
+## Optional secondary options
+
+### `exceptWithoutPropertyFallback`: `["array", "of", "unprefixed", /functions/, "/regex/"]|"function"|"/regex/"|/regex/`
+
+Requires specified functions to have a fallback property declaration before them within the same selector.
+The function must be included in the primary option list to be affected by this option.
+
+Given:
+
+```json
+["min", "/max/"]
+```
+
+The following patterns are considered problems:
+
+<!-- prettier-ignore -->
+```css
+a { width: min(50%, 100px);}
+```
+
+<!-- prettier-ignore -->
+```css
+a { height: max(50%, 100px); }
+```
+
+<!-- prettier-ignore -->
+```css
+a {
+  width: max(50%, 100px);
+  width: 100px;
+}
+```
+
+The following patterns are _not_ considered problems:
+
```

Comment:
Let's add an example for `a { transform: scale(1); }` here to match the new description above.